Test/git-issues/git-issue-779.dfy

```diff
@@ -0,0 +1,33 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+// Lines marked BOGUS were problems before this bug was fixed
```

Comment:
I suggest you change "BOGUS" below to "regression". Please also change the verbs to past tense, e.g., change "gives" to "gave" or "once gave" or "used to give".

Reply:
Done